aws_jobs/cyhy-data-extract.py

```diff
@@ -385,20 +385,7 @@ def main():
         # only pull the entire collection every time an extract is run.
         "requests": {
             "query": {},
-            "projection": {
-                "agency.acronym": True,
-                "agency.location": True,
-                "agency.name": True,
-                "agency.type": True,
-                "children": True,
-                "enrolled": True,
-                "networks": True,
-                "period_start": True,
-                "report_types": True,
-                "retired": True,
-                "scan_types": True,
-                "stakeholder": True,
-            },
+            "projection": {},
```

Comment:
I'd like to see a comment here to warn future readers that this empty projection dumps all fields, some of which may contain sensitive data.
```suggestion
            # Note that this empty projection exports all fields, some
            # of which may contain sensitive data.
            "projection": {},
```